harness/determined/tensorboard/metric_writers/pytorch.py

```diff
@@ -12,7 +13,7 @@
 from torch.utils.tensorboard import SummaryWriter  # isort:skip
 
 
-class TorchWriter(tensorboard.MetricWriter):
+class _TorchWriter(tensorboard.MetricWriter):
     """
     TorchWriter uses PyTorch file writers and summary operations to write
```

Comment:
leave the user-facing docstring attached to the user-facing symbol (that is, `TorchWriter` not `Torchwriter`).

Also please add a warning admonition to the docstring.  [Here's an example](https://github.com/determined-ai/determined/blob/master/harness/determined/common/experimental/checkpoint/_checkpoint.py#L293-L300) from the last time I did this.

Notice how detailed the docstring I wrote was.  I believe these sorts of corner-case runtime messages are excellent places for clear documentation.  Unlike the general docs site, where you don't necessarily know who's reading your documentation or what they're looking for, when you write a runtime message like this you know _exactly_ what your reader needs to know right now.